dockers/docker-fpm-frr/frr/zebra/zebra.conf.j2

```diff
@@ -6,6 +6,13 @@
 !
 {% endblock banner %}
 !
+{% block fpm %}
+! Uses the old known FPM behavior of including next hop information in the route (e.g. RTM_NEWROUTE) messages
```

Comment:
What is the behavior if the user has split mode and performs an upgrade?

Reply:
User needs to update his own configs